config/locales/idv/en.yml

```diff
@@ -82,10 +82,10 @@ en:
         warning: Please check the information you entered and try again. Common mistakes
           are an incorrect Social Security number or ZIP Code.
       setup:
-        fail_html: Please %{contact_form_link} and enter the code <b>%{support_code}</b>
-          in the detail field.
-        heading: There is an issue with your information
-        link_text: fill out our contact form
+        fail_html: 'Please call our contact center at <strong>(844) 875-6446</strong> to
```

Comment:
I think it could be wise to make this configurable or at least constant-ized, with a placeholder value in the default configuration. For example, I expect we wouldn't want people using this number if they were testing in a lower sandbox environment.

Reply:
Yeah I was mulling that over that yesterday ([here](https://gsa-tts.slack.com/archives/C0NGESUN5/p1671656789362999)). It's unclear to me how this live call part of things gets tested. That said, I think you're right.

Reply:
Yeah this phone number should probably be stored in the IdentityConfig.store